internal/platform/crypto.go

```diff
@@ -0,0 +1,17 @@
+package platform
+
+import (
+	"io"
+	"math/rand"
+)
+
+// seed is a fixed seed value for NewFakeRandSource.
+//
+// Trivia: While arbitrary, 42 was chosen as it is the "Ultimate Answer" in
```

Comment:
a bit wordy, but I suspect not everyone knew why we have 42 in popping up in source files